### PR TITLE
Alvaro's AD review comments

### PR DESCRIPTION
**Pull request by martinthomson**

### CODE REVIEW

File: draft-ietf-git-using-github.md [hunk 1]
```diff
@@ -620,9 +630,9 @@ issues as they see fit.
 As documents mature further, explicit confirmation of technical decisions with
 the Working Group mailing list becomes more important.
 
-Gaining Working Group consensus about the resolution of issues can be done in
-the abstract, with editors being permitted to capture the outcome of discussions
-as they see fit.
+Chairs are able to declare Working Group consensus about the resolution of
```
Comment (chris-wood) [inline draft-ietf-git-using-github.md]:
```suggestion
Chairs can declare Working Group consensus about the resolution of
```

File: draft-ietf-git-using-github.md [hunk 2]
```diff
@@ -23,6 +23,13 @@ author:
    email: barbara.stark@att.com
 
 informative:
+  GLOSSARY:
+    title: "GitHub glossary"
+    date: 2020-03
+    target: "https://help.github.com/en/github/getting-started-with-github/github-glossary"
```
Comment (chris-wood) [inline draft-ietf-git-using-github.md:29]: Are we concerned about this link not aging well? (I wonder if we should, in parallel, try to move the contents of https://ietf-gitwg.github.io to some place more stable and under control of the IETF.)

**Review by chris-wood — APPROVED** [review]

File: draft-ietf-git-using-github.md (hunk 2 above)
Comment (martinthomson) [reply draft-ietf-git-using-github.md:29]: Yeah, but I'm not sure that our own copy of this is guaranteed to be any better.  The GitHub link is nice in the sense that it concentrates on terminology.

Comment (chris-wood) [reply draft-ietf-git-using-github.md:29]: Yep, no argument there. It certainly has better content.

**Pull request merged**